### DISCUSSION TITLE
Using cash balances in portfolio

### DISCUSSION THREAD

**Discussion by prashmohan** [ghostfolio/ghostfolio#2324] [Q&A]
Related to the discussion in https://github.com/ghostfolio/ghostfolio/discussions/2308, the portfolio tab of ghostfolio also does not include the cash holdings. Cash should be considered an asset just as stock holdings are. In effect the portfolio and summary should show the net worth instead of the value of security holdings.

A specific example in my case. I have a negative cash reserve because of a margin loan. Not including this "loan" basically inflates the value of the holdings and does not reflect the liquidity value of the account.

**Comment by dtslvr** [comment]
Hello @prashmohan

I agree with you that cash is an essential part of the whole picture. At the moment it is not taken into account under _Portfolio_ > _Analytics_. Only recently, the cash balance has been [extended to support a time series](https://github.com/ghostfolio/ghostfolio/pull/2166). In my opinion it would be interesting to transform the _Portfolio Evolution_ chart into a _Net Worth_ chart, so that the cash balances of the accounts are considered (or do we still need the original _Portfolio Evolution_ chart in addition?). Would you be interested in making a contribution?

Regarding _Summary_, the cash holdings and liabilities are already taken into account in the _Net Worth_ calculation, aren't they?

**Reply by prashmohan** [reply]
You are absolutely right. The summary tab does take it into account.

The time series nature of cash is exactly what I was referring to. I don't personally have a need for a portfolio holdings chart that is different from a net worth chart.

> Would you be interested in making a contribution?

I can take a look over the weekend. Albeit I have very little front end experience 🙂

**Reply by dtslvr** [reply]
> I can take a look over the weekend. Albeit I have very little front end experience 🙂

You can fully focus on the endpoint and I'm happy to do the front-end integration. I have created a ticket here: https://github.com/ghostfolio/ghostfolio/issues/2328

Does the description make sense to you? Please comment directly in the ticket and I will assign you.
